src/components/Reports/ReportEdit.tsx

```diff
@@ -39,8 +43,20 @@ export default function ReportEdit(): JSX.Element {
     } else {
       snackbar.toastError(strings.GENERIC_ERROR, strings.REPORT_COULD_NOT_OPEN);
     }
+
+    setCheckUserInterval(setInterval(getReport, 60000));
```

Comment:
do we need to return a () => clearInterval ? when component dismounts?
something like that.. so we aren't polling forever

Reply:
I changed the implementation because using useState was causing more than a interval at the same time. Modified to use useRef and added a clean up